enhancements/user-monitoring.md

```diff
@@ -0,0 +1,248 @@
+---
+title: user-monitoring
+authors:
+  - "@s-urbaniak"
+  - "@brancz"
+reviewers:
+  - "@sichvoge"
+  - "@openshift/openshift-team-monitoring"
+approvers:
+  - "@sichvoge"
+  - "@openshift/openshift-team-monitoring"
+creation-date: 2019-09-19
+last-updated: 2019-09-19
+status: implementable
+---
+
+# user-monitoring
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+This enhancement describes a new feature for OpenShift cluster observability, namely user monitoring.
+It is an extension to the existing cluster monitoring stack to enable observability of user namespaces.
+Currently, the cluster monitoring stack only reconciles OpenShift namespaces.
+
+## Motivation
+
+Currently there is no supported solution for customers to monitor their business critical applications.
+The existing cluster monitoring stack has no support for adding scrape targets in user namespaces
+as it is only available for OpenShift components scraping workloads from `openshift-*` namespaces.
+Other Red Hat products, outside the OpenShift framework components
+don’t have a monitoring solution to integrate their metrics and out-of-the-box insights into their well being.
+Currently it is difficult to achieve metering/chargeback for Red Hat products without a single source.
+
+### Goals
+
+Allow deployment and reconcilation scrape targets, recording rules, and alerting rules in user namespaces.
+This effectively allows the reconcilation of the following custom resources in user namespaces:
+
+- ServiceMonitor (https://github.com/openshift/prometheus-operator/blob/master/Documentation/api.md#servicemonitor)
+- PodMonitor (https://github.com/openshift/prometheus-operator/blob/master/Documentation/api.md#podmonitor)
+- PrometheusRule (https://github.com/openshift/prometheus-operator/blob/master/Documentation/api.md#prometheusrule)
+
+### Non-Goals
+
+Out of scope are:
+- Definition of Dashboards.
+- Deployment of Prometheus instances in user namespaces.
+- Deployment of Alertmanager instances user namespaces.
+
+## Proposal
+
+![](user-monitoring-deployment.png)
+
+The existing cluster monitoring stack is going to be extended with additional Prometheus (P) servers
+and one additional Prometheus Operator (PO) reconciling user namespaces
```

Comment:
why a new operator?  can the existing operator not do the additional work?  is there an advantage to having it as a separate operator?  (it's still going to be CVO managed, right?)